experiments/run_landcoverai_seed_experiments.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Runs the train script with a grid of hyperparameters."""
+import itertools
+import os
+import subprocess
+from multiprocessing import Process, Queue
+
+# list of GPU IDs that we want to use, one job will be started for every ID in the list
+GPUS = [3, 4, 5, 6, 7]
+DRY_RUN = False  # if False then print out the commands to be run, if True then run
+DATA_DIR = ""
+
+# Hyperparameter options
+model_options = ["unet"]
+encoder_options = ["resnet50"]
+lr_options = [1e-4]
+loss_options = ["ce"]
+weight_init_options = ["imagenet"]
+
+seeds = list(range(15))
+
+
+def do_work(work: "Queue[str]", gpu_idx: int) -> bool:
+    """Process for each ID in GPUS."""
+    while not work.empty():
+        experiment = work.get()
+        experiment = experiment.replace("GPU", str(gpu_idx))
+        print(experiment)
+        if not DRY_RUN:
+            subprocess.call(experiment.split(" "))
+    return True
+
+
+if __name__ == "__main__":
+    work: "Queue[str]" = Queue()
+
+    for (model, encoder, lr, loss, weight_init, seed) in itertools.product(
+        model_options,
+        encoder_options,
+        lr_options,
+        loss_options,
+        weight_init_options,
+        seeds,
+    ):
+
+        experiment_name = f"{model}_{encoder}_{lr}_{loss}_{weight_init}_{seed}"
+
+        output_dir = "output/landcoverai_seed_experiments/"
```

Comment:
The `os.path.join` stuff applies to other files too

Reply:
think I got it this time